tests/e2e/matmul/CMakeLists.txt

```diff
@@ -1741,6 +1741,36 @@ iree_generated_e2e_runner_test(
     "requires-gpu-cdna3"
 )
 
+iree_generated_e2e_runner_test(
+  NAME
+    e2e_matmul_cdna3_dt_f64
```

Comment:
I'm confused that a cdna3 test suite is added but the PR title is about CDNA1/2. Why?

Reply:
Sorry, that's just a mishap while slicing my diff into PRs. This was meant to be part of https://github.com/iree-org/iree/pull/19099.

Reply:
I see, can you rebase the PR (or merge the main branch into your branch)?

Reply:
@hanhanW , if I do that, I will lose the green CI status. The bottom commit here is already in main.  Can you review based on looking at the top commit?

Reply:
ah, I see. I did not notice that there are two commits. Reviewed and it looks good to me, thanks!